addons/web_editor/static/lib/odoo-editor/test/utils.js

```diff
@@ -287,6 +287,7 @@ export function renderTextualSelection() {
 
 export async function testEditor(Editor = OdooEditor, spec) {
     const testNode = document.createElement('div');
+    document.querySelector('#editor-test-container').innerHTML = '';
```

Comment:
Why is this needed now ?

Reply:
Sometimes some tests are affected if the previous Editor instances and it's editable DIV are not completely and properly cleaned. 
e.g. test **green** if run alone, but **red** in bulk.

No idea why, but I added that to make sure it didn't append.